interfaces/builtin/pluggable_storage.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const pluggableStorageConnectedPlugAppArmor = `
+# Description: Can access removable storage filesystems
+
+# Filesystems are mounted to /run/media/<user>/foo since /media is read-only
+/run/media/** rw,
```

Comment:
Re testing> snap-confine is in recent ubuntu-core snaps. I've not used the new udf yet but https://code.launchpad.net/~snappy-dev/+snap/ubuntu-core has builds which show, for example:

Get:2 http://ppa.launchpad.net/snappy-dev/image/ubuntu xenial/main armhf snap-confine armhf 1.0.38-0ubuntu0.16.04.7 [18.8 kB]